paper-menu-button.html

```diff
@@ -131,7 +131,7 @@
       restore-focus-on-close="[[restoreFocusOnClose]]"
       on-iron-overlay-canceled="__onIronOverlayCanceled">
       <div class="dropdown-content">
-        <content id="content" select=".dropdown-content"></content>
+        <slot id="content"></slot>
```

Comment:
[`iron-dropdown 2.0`](https://github.com/PolymerElements/iron-dropdown/pull/113/files#diff-adf398d4d0725b36fcbe886115a02754R60) doesn't enforce a slot name to ease its use by hybrid elements. This allows users to keep the old class on distributed content without specifying a slot name.

Likewise, `paper-menu-button` will first distribute `dropdown-trigger` content in the previous slot, and then the remaining content into the dropdown.

The only possible drawback of this change is that if you were distributing some element without the right class in 1.x, that will be distributed into the dropdown in 2.x, e.g.
```html
<paper-menu-button>
  <button class="dropdown-trigger" slot="dropdown-trigger">show image</button>
  <img class="dropdown-trigger" src="some-url">
  <label>super cool image</label>
</paper-menu-button>
```
^ in 1.x, you'd see only the image after clicking on the button. In 2.x you'd also see that label